models/place-video-post.json

```diff
@@ -1,5 +1,5 @@
 {
-  "title": "place.add.video",
+  "title": "place.video.post",
```

Comment:
Nit: the property name is `place.videos` so I would go with `place.videos.post` here. A POST is (in a REST API) usually to a collection, so it makes sense that its `videos` (plural) here I think.

Reply:
Also like you mentioned in the commit the format is `{offer}-{property}-{method}` and `{property}` in this case is `videos` if I'm not mistaken

Reply:
Indeed! I changed it to the plural flavour.